mike_skalandunas/gulpfile.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+var gulp = require('gulp');
+var jshint = require('gulp-jshint');
+var mocha = require('gulp-mocha');
+var appFiles = ['index.js', 'lib/**/*.js', 'bin/**/*.js'];
+var testFiles = ['./test/**/*.js'];
+var watcher = gulp.watch('./**/*.js', ['jshint:test']);
+
+watcher.on('change', function(event) {
+  console.log('File ' + event.path + ' was ' + event.type + ', running tasks...');
```

Comment:
It looks like it all works properly.  Whenever I make a change, I can see the jshint task start up.  I think I did Mocha correctly, but I'm waiting to hear Tyler talk about it.